Documentation/how-to-create-an-erd.md

```diff
@@ -0,0 +1,17 @@
+# How To create an entity relation diagram
+
+## Install Graphviz
+##### source: https://www.norbauer.com/rails-consulting/notes/erd-diagrams-from-sequel-pro.html
+```bash
+brew install graphviz
+```
+
+## Export Database from Sequel-Pro
```

Comment:
Sequel Pro*

Reply:
Export database structure* (just to make it clear that it's only the structure we're exporting and not all the data as well)